### PR TITLE
Some small doc fixes

### PR DESCRIPTION
**Pull request by md-magenta**
I have made a few small doc fixes as I worked my way through the tutorial.

### CODE REVIEW

File: doc/REST_API_getting-started.rst [hunk 1]
```diff
@@ -71,8 +72,7 @@ This can take a while...
 Once the installation process is complete,
 you may confirm that the OIO rest api is running::
 
-  $ apt install curl
-  $ curl http://localhost:8080
+  $ curl -l http://localhost:8080
```
Comment (carlbordum) [inline doc/REST_API_getting-started.rst]: big "L"?

Comment (md-magenta) [reply doc/REST_API_getting-started.rst]: fixed

File: doc/REST_API_getting-started.rst [hunk 2]
```diff
@@ -24,11 +25,12 @@ Attach to the container by doing::
...
-Add a unix user account: ::
+Add a unix user account and install some utilities: ::
 
   $ adduser operator
   $ addgroup operator sudo
-  $ apt install ssh
+  $ apt update
+  $ apt install ssh git curl
 
 Lookup the IP-address of the container::
```
Comment (agger-magenta) [inline doc/REST_API_getting-started.rst]: "IP address" er i to ord på engelsk, nu vi er ved det.

File: doc/REST_API_getting-started.rst [hunk 3]
```diff
@@ -91,9 +91,7 @@ which should give a JSON response like::
 Getting to know LoRas REST API
 ------------------------------
 The following small exercises can be used as an inspiration to getting to know
-LoRas REST API. Read the
-`HOWTO <https://github.com/magenta-aps/mox/blob/master/doc/LoRa_HOWTO.rst>`_
-from the LoRa GitHub page before moving on. Also, have a look at the LoRa
+LoRas REST API. Read :ref:`LoRaHOWTO` before moving on. Also, have a look at the LoRa
```
Comment (agger-magenta) [inline doc/REST_API_getting-started.rst]: LoRa's

Comment (danchr) [reply doc/REST_API_getting-started.rst]: > LoRa's

Actually, I'd say the construction is correct as-is; if you want to add an apostrophe, you need get rid of the “the”.

Comment (agger-magenta) [reply doc/REST_API_getting-started.rst]: * "the LoRa REST API" :+1: 
* "LoRa's REST API" :+1: 
* "LoRas REST API" :-1:

Comment (danchr) [reply doc/REST_API_getting-started.rst]: > > LoRa's
> 
> Actually, I'd say the construction is correct as-is; if you want to add an apostrophe, you need get rid of the “the”.

Never mind — I misunderstood!
```suggestion
LoRa's REST API. Read :ref:`LoRaHOWTO` before moving on. Also, have a look at the LoRa
```

**Comment by md-magenta** [comment]
Superseded by #103

**Pull request closed without merging**